ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/DbChangelog/Update.hs

```diff
@@ -416,6 +434,12 @@ data DiffsToFlush l = DiffsToFlush {
 
 -- | "Flush" the 'DbChangelog' by splitting it into the diffs that should be
 -- flushed and the new 'DbChangelog'.
+--
+-- >       1     | [ L3, L4, L5, L6 ] | [ D2, D3, D4, D5, D6 ]
+-- >>> splitForFlushing
+-- > ( (2, [ D2, D3 ])
+-- > , (3, [ L3, L4, L5, L6 ], [ D4, D5, D6 ])
+-- > )
```

Comment:
I'm assuming `2` is `toFlushSlot` here, but I'm not entirely sure. And shouldn't it be `3` in that case?

Reply:
Possibly, we should also use the same notation for the `rdblog` part as you've been using throughout the other examples. i.e., `| .* | .* | .* |`